src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj

```diff
@@ -5,6 +5,14 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Can you delete these.

Reply:
This intrigues me. I noticed Dave had added this exclusion to the Functions Acceptance tests and they were working on my machine without the continual deletion of the '**' files. So although I really don't like it, does it cause us a problem leaving it in?